### DISCUSSION TITLE
HA or cluster setup?

### DISCUSSION THREAD

**Discussion by mntnclmbr** [halfgaar/FlashMQ#29] [Q&A] [answered]
I saw HAProxy support was being added, how would you suggest to create high availability clusters with flashmq? If a publisher is on one machine and the corresponding subscriber on another, will that still work ? Should the individual flashmq brokers have an inter connect via a bridge config or something like that ?

**Comment by halfgaar** [comment, marked as answer]
The haproxy protocol support was added because it's easy. It allows you to put a haproxy in front it and still read the original client address. But actual clustering would indeed need support from FlashMQ. It would need some kind of back-haul between servers; a bridge isn't enough.

It's definitely on my mind to add support for this, but it will take some effort.
